book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Please check out the example below ⬇️  Please implement it in your code to make it completed 👍 
 ```
def add_rental(person, date)
    Rental.new(date, self, person)
 end

```

Reply:
Thanks for your insights @codecaiine, I will take care of it too 😉 .